### PR TITLE
Added cmake file for ios-nocodesign-12-1

### PR DESCRIPTION
**Pull request by casparjespersen**

### CODE REVIEW

File: ios-nocodesign-12-1-dep-9-0-bitcode-cxx11.cmake [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# Copyright (c) 2017, Ruslan Baratov
+# All rights reserved.
+
```
Comment (ruslo) [inline ios-nocodesign-12-1-dep-9-0-bitcode-cxx11.cmake]: Please remove trailing whitespaces.

**Comment by cguentherTUChemnitz** [comment]
Why is this not merged yet?

File: ios-nocodesign-12-1-dep-9-0-bitcode-cxx11.cmake (hunk 1 above)
Comment (casparjespersen) [reply ios-nocodesign-12-1-dep-9-0-bitcode-cxx11.cmake]: I don't see any trailing whitespaces. Please elaborate.

Comment (ruslo) [reply ios-nocodesign-12-1-dep-9-0-bitcode-cxx11.cmake]: > I don't **see** any trailing whitespaces

You can't see them, they are whitespaces :)

> Please elaborate

Check this commit: https://github.com/ruslo/polly/pull/283/commits/f114519882896736e8edc96910f494834a792547

**Pull request merged**